src/backend/drm/compositor/gbm.rs

```diff
@@ -185,14 +194,19 @@ pub struct DrmError {
 }
 
 /// Attach a [`framebuffer::Handle`] to an [`BufferObject`]
-pub fn framebuffer_from_bo<T>(drm: &DrmDeviceFd, bo: &BufferObject<T>) -> Result<GbmFramebuffer, DrmError> {
+pub fn framebuffer_from_bo<T>(
+    drm: &DrmDeviceFd,
+    bo: &BufferObject<T>,
+    consider_opaque: bool,
```

Comment:
As this actually implements a fallback path we may want to also express that in the argument name.
Maybe something like `allow_opaque_fallback`?

Reply:
done